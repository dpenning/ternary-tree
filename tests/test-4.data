(23	52	12	12	23
43	78	34	37	48
87	95	60	57	52
22	17	84	6	90
35	3	8	13	22
91	89	38	74	8
97	47	40	41	22
85	10	21	100	25
21	94	3	100	74
2	12	65	83	62
4	77	29	92	15
1	95	71	84	62
46	88	83	56	82
30	22	64	33	14
49	49	49	62	32
37	38	83	16	86
96	82	29	27	26
9	30	80	19	17
90	45	80	24	68
59	84	38	82	95)