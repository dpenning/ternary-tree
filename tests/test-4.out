
(((((1) 2 NIL 3 NIL) 3 (4) 6 (NIL 8 NIL 8 ((9) 10 NIL 12 NIL))) 12 NIL 12
  ((NIL 13 (NIL 14 NIL 15 (16 17)) 17 (((19) 21 NIL 21 (22)) 22 NIL 22 NIL)) 22
   NIL 23 NIL))
 23
 (((24) 25 ((26) 27 NIL 29 NIL) 29 ((30) 30 (32) 33 NIL)) 34
  (NIL 35 (37) 37 ((38 38) 38 NIL 40 (41))) 43
  (((45) 46 NIL 47 NIL) 48 ((49) 49 NIL 49 NIL) 52 NIL))
 52
 (((56) 57 (59) 60 (((62 62) 62 (64) 65 (68 71)) 74 NIL 74 (77))) 78
  (((((80) 80 (82) 82 NIL) 82 (83) 83 NIL) 83 (84) 84 NIL) 84 NIL 85 (86)) 87
  ((NIL 88 NIL 89 (90)) 90 (NIL 91 (92) 94 (95 95)) 95
   ((96) 97 (100) 100 NIL)))) 
